### PR TITLE
Allow starting a videochat within groups

### LINKED ISSUE #2222: Start a videochat is not available for groups
The option in the menu to send a videochat invitation does not appear in groups.
As the videochat itself is handled externally, I don't see a reason why DeltaChat should not enable this button for groups.

- Operating System (Linux/Mac/Windows/iOS/Android): Windows
- Delta Chat Version: 1.15.4
- Expected behavior: Menu for a chat includes "videochat" even for groups
- Actual behavior: Menu only contains the option "videochat" for 1 to 1 chats
- Steps to reproduce the problem: Open a group chat, click on the 3 dot button in the top left corner and do not see the videochat entry
- Screenshots: 

Menu from a 1-to-1 chat with the videochat entry:
grafik

- Source:

https://github.com/deltachat/deltachat-desktop/blob/master/src/renderer/components/Menu.tsx#L124

### PR DESCRIPTION
**Pull request by AUtech-dev**
Fixes #2222

### CODE REVIEW

**Comment by AUtech-dev** [comment]
https://github.com/deltachat/deltachat-desktop/pull/2223/checks?check_run_id=2352454703 shows a lot of test errors. But as far as I see I didn't change anything in this files.

Can somebody help me?

**Comment by Simon-Laux** [comment]
don't worry about the integration tests, they are currently broken, also on master.

File: src/renderer/components/Menu.tsx [hunk 1]
```diff
@@ -68,10 +68,7 @@ export default function DeltaMenu(props: { selectedChat: FullChat }) {
     })
   const onVideoChat = () => {
     screenContext.openDialog('ConfirmationDialog', {
-      header: tx(
-        'videochat_invite_user_to_videochat',
-        selectedChat.contacts[0]?.displayName
-      ),
+      header: tx('videochat_invite_user_to_videochat', selectedChat.name),
```
Comment (Simon-Laux) [inline src/renderer/components/Menu.tsx:71]: maybe another translation string is needed... 
"Invite %1$s to a video chat?" might work for now, but as the key ('videochat_invite_user_to_videochat') says its inviting a user I don't know if this still works in all translations, what do you think @r10s?

**Comment by adbenitez** [comment]
equivalent PR for Android: https://github.com/deltachat/deltachat-android/pull/1873

File: src/renderer/components/Menu.tsx (hunk 1 above)
Comment (adbenitez) [reply src/renderer/components/Menu.tsx:71]: actually, android uses chat.getName() already, so there is no issue in using the chat name, it shows:

"Invite MyGroup to a videochat?" which is fine

**Review by adbenitez — APPROVED** [review]
lgtm, thanks!

**Pull request merged**